web-app/src/app/tabs/faults/faults.component.html

```diff
@@ -1,5 +1,130 @@
-<div class="faults container-column">
-	<div class="container-row">
+<div class="faults">
+<div class="container-row full">
+	<div class="leftside">
+			<div class="subheader">Battery Faults</div>
+			<div class="item off" [ngClass]="{'on' : batteryFaults.errorFlags.alwaysOnSupplyFault == true, 'off' : batteryFaults.errorFlags.alwaysOnSupplyFault == false}">Always On Supply Fault</div>
+			<div class="item off" [ngClass]="{'on' : batteryFaults.errorFlags.canbusCommunicationsFault == true, 'off' : batteryFaults.errorFlags.canbusCommunicationsFault == false}">CANBUS Communications Fault</div>
```

Comment:
You can remove the true and false and just put batteryFaults... and !batteryFaults.... Also I don't know if setting it to off by default should make checking for false necessary. I noticed in my code I did a bit of both so I might have to fix mine. Hopefully you can experiment and let me know.

Reply:
Not sure, but if we don't check if it's false, wouldn't it be unable to go back to being false?